client/src/api/histories.ts

```diff
@@ -8,3 +8,4 @@ export const undeleteHistory = fetcher.path("/api/histories/deleted/{history_id}
 export const undeleteHistories = fetcher.path("/api/histories/batch/undelete").method("put").create();
 export const publishedHistoriesFetcher = fetcher.path("/api/histories/published").method("get").create();
 export const historyFetcher = fetcher.path("/api/histories/{history_id}").method("get").create();
+export const putHistories = fetcher.path("/api/histories/{history_id}/contents/bulk").method("put").create();
```

Comment:
`putHistories` sounds too general, the name should tell something about "bulk" updating history items. Maybe `updateHistoryItemsInBulk`?